scripts/functions/gemset

```diff
@@ -355,6 +355,7 @@ gemset_initial()
 run_gem_wrappers()
 {
   gem_install gem-wrappers >/dev/null &&
+  gem pristine gem-wrappers --only-plugins >/dev/null &&
```

Comment:
We should only do this if `gem wrappers >/dev/null` fails first, right? Otherwise it will do the `pristine` when it's already pristine. This command is run more than once, so we don't want to do `gem pristine` if we don't have to.

Reply:
This comment still applies.